src/qibolab/platform/platform.py

```diff
@@ -328,26 +334,26 @@ def dump(self, path: Path):
         """Dump platform."""
         (path / PARAMETERS).write_text(self.parameters.model_dump_json(indent=4))
 
-    def get_qubit(self, qubit: QubitId) -> Qubit:
-        """Return the name of the physical qubit corresponding to a logical
-        qubit.
+    def _element(self, qubit: QubitId, coupler=False) -> tuple[QubitId, Qubit]:
+        elements = self.qubits if not coupler else self.couplers
+        try:
+            return qubit, elements[qubit]
+        except KeyError:
+            assert isinstance(qubit, int)
+            return list(self.qubits.items())[qubit]
```

Comment:
Shouldn't this be
```suggestion
        try:
            return qubit, elements[qubit]
        except KeyError:
            assert isinstance(qubit, int)
            name = list(elements)[qubit]
            return name, elements[name]
```
? I think as it is now the `except` branch doesn't agree with the return type (and the `try` branch).

Also, not related to this PR, but at some point we could move this to the compiler, since, if I remember correctly, it is only used there. It is a side-effect of the fact that in Qibo we always use integers to enumerate qubits, while in qibolab we allow for string names.

Reply:
> ? I think as it is now the `except` branch doesn't agree with the return type (and the `try` branch).

My type checker is telling that is fine, without having to do it manually :)

The trick is that `.items()` is producing an iterable of 2-tuples, so `list(x.items())` is a list which elements are exactly what I want to return.